Modules/socketmodule.c

```diff
@@ -305,6 +305,69 @@ if_indextoname(index) -- return the corresponding interface name\n\
 /* Provides the IsWindows7SP1OrGreater() function */
 #include <VersionHelpers.h>
 
+/* This block removes some flags on older version Windows during run-time.
+   https://msdn.microsoft.com/en-us/library/windows/desktop/ms738596.aspx */
+#if 1
```

Comment:
No need for the `#if 1` (I assume this is already within an `MS_WINDOWS` block)

Reply:
I thought some IDEs can fold this code block, I removed it.